bindings/java/src/main/com/apple/foundationdb/NativeObjectWrapper.java

```diff
@@ -45,18 +45,6 @@ public boolean isClosed() {
 		return closed;
 	}
 
-	public void checkUnclosed(String context) {
-		try {
-			if(FDB.instance().warnOnUnclosed && !closed) {
-				System.err.println(context + " not closed");
```

Comment:
I actually like that error msg. Can we jsut keep the error msg but remove all the rest - remove the what ever actual cleanup we do (if we do)?

Reply:
I don't think so, because the error message relies on finalizers that we don't want to have.

Reply:
See https://github.com/apple/foundationdb/issues/3190